### PR TITLE
make withscores optional in getting events query

### PR DESCRIPTION
**Pull request by jimyang2008**
https://github.com/firewalla/firecommit/issues/118#issuecomment-766497318

### CODE REVIEW

File: scripts/event_collectors/service_status.sh [hunk 1]
```diff
@@ -20,17 +20,69 @@
...
+    check_each_system_service brofish "active" || _rc=1
+    check_each_system_service firewalla "inactive" || _rc=1
+    check_each_system_service fireupgrade "inactive" || _rc=1
+    check_each_system_service fireboot "inactive" || _rc=1
+
+    vpn_state=$(redis-cli hget policy:system vpn | jq .state)
+    $vpn_state && vpn_run_state='active' || vpn_run_state='inactive'
+    check_each_system_service openvpn@server $vpn_run_state || _rc=1
+
+    if [[ $PLATFORM != 'gold' ]]; then # non gold
```
Comment (MelvinTo) [inline scripts/event_collectors/service_status.sh:49]: use env MANAGED_BY_FIREROUTER, do not hard code gold here

File: scripts/sanity_check.sh [hunk 2]
```diff
@@ -130,15 +130,18 @@ check_systemctl_services() {
...
     check_each_system_service firekick "dead"
     check_each_system_service redis-server "running"
-    check_each_system_service openvpn@server "running"
-    check_each_system_service watchdog "running"
     check_each_system_service brofish "running"
     check_each_system_service firewalla "dead"
     check_each_system_service fireupgrade "dead"
     check_each_system_service fireboot "dead"
 
+    vpn_state=$(redis-cli hget policy:system vpn | jq .state)
```
Comment (MelvinTo) [inline scripts/sanity_check.sh:138]: just be aware that jq is not available in old Red/Blue

File: scripts/event_collectors/service_status.sh [hunk 3]
```diff
@@ -20,17 +20,69 @@
...
+    check_each_system_service firemain "active" || _rc=1
+    check_each_system_service firemon "active" || _rc=1
+    check_each_system_service firekick "inactive" || _rc=1
+    check_each_system_service redis-server "active" || _rc=1
+    check_each_system_service brofish "active" || _rc=1
+    check_each_system_service firewalla "inactive" || _rc=1
+    check_each_system_service fireupgrade "inactive" || _rc=1
+    check_each_system_service fireboot "inactive" || _rc=1
+
+    vpn_state=$(redis-cli hget policy:system vpn | jq .state)
```
Comment (MelvinTo) [inline scripts/event_collectors/service_status.sh:45]: jq not available in Red/Blue

Comment (jimyang2008) [reply scripts/event_collectors/service_status.sh:45]: replaced with `fgrep`

File: scripts/sanity_check.sh (hunk 2 above)
Comment (jimyang2008) [reply scripts/sanity_check.sh:138]: replaced with `fgrep`

File: scripts/event_collectors/service_status.sh (hunk 1 above)
Comment (jimyang2008) [reply scripts/event_collectors/service_status.sh:49]: fixed as suggested

**Review by MelvinTo — APPROVED** [review]

**Pull request merged**